src/libraries/System.Numerics.Tensors/src/ReferenceAssemblyExclusions.txt

```diff
@@ -0,0 +1,2 @@
+M:System.Numerics.Tensors.TensorPrimitives.ConvertToHalf(System.ReadOnlySpan{System.Single},System.Span{System.Half})
+M:System.Numerics.Tensors.TensorPrimitives.ConvertToSingle(System.ReadOnlySpan{System.Half},System.Span{System.Single})
```

Comment:
Why are these excluded?

Reply:
They can't go in the ref because `System.Half` doens't exist in netstandard 2.0/framework.

Reply:
If they aren’t in the ref, they aren’t available on net8.0 either.

they likely need to be conditioned in the ref so they exist for net8.0 and not for netstandard2.0

Reply:
They're already in a separate file that's already conditionally compiled in. I assume Michael is doing this so that when he runs the generator, it doesn't try to add them to the main file?

Reply:
Thats correct, this prevents them from being added to the main file.